src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
 .catalog {
-  display: flex;
+  display: grid;
+  grid-template: 300px / repeat(4, 200px);
   flex-wrap: wrap;
-
+  gap: 48px;
+  justify-content: center;
+  place-items: center;
   max-width: 944px;
+  padding-bottom: 48px;
   margin: 0 auto;
 
+  @include media(1024px, 3);
+  @include media(768px, 2);
+  @include media(488px, 1);
```

Comment:
According to the task, four columns starting at 1024px, but you have 3, the same situation with 768, 488.
![image](https://user-images.githubusercontent.com/119349933/208306162-d6b401f9-d023-41c2-8b90-066e6e143522.png)

Reply:
fix @OlyaZaiets comment